frontend/src/components/DetailsTable.tsx

```diff
@@ -40,43 +42,60 @@ export const css = stylesheet({
   },
   valueText: {
     maxWidth: 400,
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
   },
 });
 
 interface DetailsTableProps {
-  fields: string[][];
+  fields: Array<KeyValue<string|S3Artifact>>;
   title?: string;
+  valueComponent?: React.FC<S3Artifact>;
```

Comment:
Can you add a simple test for this field in `DetailsTable.test.tsx`?

Reply:
ok done.